okio/src/commonTest/kotlin/okio/PathTest.kt

```diff
@@ -380,4 +381,131 @@ class PathTest {
     assertEquals("a".toPath().parent!!, "a".toPath().parent!!)
     assertEquals("a/b".toPath().parent!!, "a\\b".toPath().parent!!)
   }
+
+  @Test
+  fun relativeToWindowsPaths() {
+    val a = "C:\\Windows\\notepad.exe".toPath()
+    val b = "C:\\".toPath()
+    assertEquals("..\\..".toPath(), b.relativeTo(a))
+    assertEquals("Windows\\notepad.exe".toPath(), a.relativeTo(b))
+
+    val c = "C:\\Windows\\".toPath()
```

Comment:
this test is a bit impossible because `c` won’t have the trailing `/` after toPath(). (But keep it, I like it)